Assets/Scripts/SS3D/UI/Settings/UserInterfaceSettings.cs

```diff
@@ -0,0 +1,38 @@
+using UnityEngine;
+using UnityEngine.Assertions;
+using Coimbra;
+using SS3D.Networking.Debug;
+using SS3D.Networking.Settings;
+
+namespace SS3D.UI.Settings
+{
+    public class UserInterfaceSettings : MonoBehaviour
```

Comment:
I don't understand why this is needed, we already have a ScriptableSettings to add the bandwidth display. I also don't agree at all with having stamina and bandwidth be in the same script in any form, they are not related and if we want to add more stuff it will quickly become a gigantic class.

I really don't see the need for this class at all